scripts/reborn_webui_v2_live_qa/case_matrix.py

```
@@ -0,0 +1,239 @@
+"""Static case metadata for the Reborn WebUI v2 live QA lane."""
+
+from __future__ import annotations
+
+import re
+from typing import Any, Awaitable, Callable
```

Ruff (0.15.18)

[warning] Import from `collections.abc` instead: `Awaitable`, `Callable`

Import from `collections.abc`

(UP035)